toolparameteriser/testresults.py

```diff
@@ -52,14 +53,18 @@ def clean(dct):
     return dct
 
 
-def get(completed_jobs:str,results_path,use_GPUs:bool=True):
+def get(completed_jobs:str,results_path,use_GPUs:bool=True,debug:bool=False):
+
+    toolparameteriser.utils.setlogging(debug)
```

Comment:
As above. Why does this need to also set the logging?

Reply:
Removed.